src/i18n.js

```diff
@@ -1,18 +1,17 @@
-/* eslint-disable import/no-unresolved */
 import i18n from 'i18n-js';
 
-import English from '@/languageBase/en.json';
-import Hindi from '@/languageBase/hi.json';
-import Nepali from '@/languageBase/ne.json';
-import Bengali from '@/languageBase/be.json';
-import Urdu from '@/languageBase/ur.json';
-import Assamese from '@/languageBase/as.json';
-import Gujarati from '@/languageBase/gu.json';
-import Kannada from '@/languageBase/kn.json';
-import Kashmiri from '@/languageBase/ks.json';
-import Konkani from '@/languageBase/ko.json';
-import Maithili from '@/languageBase/ma.json';
-import Malayalam from '@/languageBase/ml.json';
+import English from '../translations/en.json';
```

Comment:
why do you need `../`? Are you facing an error with `@/`? if yes? then what is the error?